### ISSUE TITLE
create_network_from_igraph does not work for an igraph object

### ISSUE THREAD

**Issue by kozo2** [cytoscape/py4cytoscape#45]
`create_network_from_igraph` didn't work for an igraph object I created.

You can reproduce it with the code below.

```python
import pandas as pd
from igraph import Graph
actors = pd.DataFrame(data={'name': ["Alice", "Bob", "Cecil", "David", "Esmeralda"],
                             'age': [48,33,45,34,21],
                             'gender': ["F","M","F","M","F"]
                             })
relations = pd.DataFrame(data={'source': ["Bob", "Cecil", "Cecil", "David", "David", "Esmeralda"],
                               'target': ["Alice", "Bob", "Alice", "Alice", "Bob", "Alice"],
                               'same_dept': [False, False, True, False, False, True],
                               'friendship': [4,5,5,2,1,1],
                               'advice': [4,5,5,4,2,3]
                               })
ig = Graph.DataFrame(relations, directed=True, vertices=actors)
p4c.create_network_from_igraph(ig, "myIgraph")
```

I got the following error with the above code.

```
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
/usr/local/lib/python3.7/dist-packages/pandas/core/indexes/base.py in get_loc(self, key, method, tolerance)
   2897             try:
-> 2898                 return self._engine.get_loc(casted_key)
   2899             except KeyError as err:

pandas/_libs/index.pyx in pandas._libs.index.IndexEngine.get_loc()

pandas/_libs/index.pyx in pandas._libs.index.IndexEngine.get_loc()

pandas/_libs/hashtable_class_helper.pxi in pandas._libs.hashtable.PyObjectHashTable.get_item()

pandas/_libs/hashtable_class_helper.pxi in pandas._libs.hashtable.PyObjectHashTable.get_item()

KeyError: 'source'

The above exception was the direct cause of the following exception:

KeyError                                  Traceback (most recent call last)
5 frames
/usr/local/lib/python3.7/dist-packages/pandas/core/indexes/base.py in get_loc(self, key, method, tolerance)
   2898                 return self._engine.get_loc(casted_key)
   2899             except KeyError as err:
-> 2900                 raise KeyError(key) from err
   2901 
   2902         if tolerance is not None:

KeyError: 'source'
```

You can reproduce the above error with the Colab link below.

https://colab.research.google.com/drive/1y_2YA2mm-wSufAk11VKpsGsXPHCmfc7k?usp=sharing

**Comment by AlexanderPico** [comment]
Ok. Narrowed it down to two issues: (1) the renaming of name to id.  This chuck (with id explicitly named) works fine:
```
actors = pd.DataFrame(data={'id': ["Alice", "Bob", "Cecil", "David", "Esmeralda"],
                             'age': [48,33,45,34,21],
                             'gender': ["F","M","F","M","F"]
                             })
relations = pd.DataFrame(data={'source': ["Bob", "Cecil", "Cecil", "David", "David", "Esmeralda"],
                               'target': ["Alice", "Bob", "Alice", "Alice", "Bob", "Alice"],
                               'same_dept': [False, False, True, False, False, True],
                               'friendship': [4,5,5,2,1,1],
                               'advice': [4,5,5,4,2,3]
                               })
p4c.create_network_from_data_frames(actors,relations)
```
And (2) the translations of the column names through igraph, as this still does not work, even with the new actors:
```
ig = Graph.DataFrame(relations, directed=True, vertices=actors)
p4c.create_network_from_igraph(ig, "myIgraph2")
```

**Comment by bdemchak** [comment]
@kozo2 @AlexanderPico This appears to be fixed in 0.0.9. I'm closing this issue, but feel free to re-open it if you find problems.
